app/controllers/pro_dashboard/open_projects_controller.rb

```diff
@@ -1,7 +1,21 @@
 class ProDashboard::OpenProjectsController < ApplicationController
+  attr_accessor :to_bid_on
   def index
     @pro = Pro.find(current_user.id)
+    @pro_location = Geokit::Geocoders::GoogleGeocoder.geocode(@pro.zipcode)
     @open_projects = @pro.open_projects
+
+    @to_bid_on = []
+
+    @open_projects.each do |open_project|
+      request_location = open_project.zipcode
+      distance = @pro_location.distance_to(request_location)
+      if distance < @pro.pro_service.radius.to_i
```

Comment:
Could use `.select` instead of `each` to pull all elements that return true
https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-select